lib/client.js

```diff
@@ -435,12 +439,23 @@ class Parser extends HTTPParser {
       return true
     }
 
+    if (headers['keep-alive']) {
+      const m = headers['keep-alive'].match(/timeout=(\d+)/)
```

Comment:
Maybe let's define the regex at the top of the file, so we don't have to generate it every time?

Reply:
> so we don't have to generate it every time?

I think v8 optimizes it out as a constant?

Reply:
I have no deep idea about V8 optimizations.

The only thing I know is just that lambdas should be faster :P

Reply:
lambdas?

Reply:
In general. I don't know anything 'bout V8 optimizes variables.